docs/discovery.md

```diff
@@ -194,7 +194,7 @@ swarm is connected to the public internet. To create your cluster:
 
     This can be on any machine or even your laptop.
 
-        $ swarm manage -H tcp://<swarm_ip:swarm_port> token://<cluster_id>
+        $ swarm manage -H <swarm_ip:swarm_port> token://<cluster_id>
```

Comment:
@allencloud : I remove the first commit. Since I am a newbie of contributing code to `Swarm`, please forgive my errors if some operations bring you in trouble, thanks!